mozetl/maudau/maudau.py

```diff
@@ -149,7 +149,7 @@ def write_locally(results):
     basename = MAUDAU_SNAPSHOT_TEMPLATE.format(publication_date)
     cols = ["day", "dau", "mau", "generated_on"]
     with open(basename, "w") as f:
-        writer = csv.DictWriter(f, cols)
+        writer = csv.DictWriter(f, cols, lineterminator="\n")
```

Comment:
The carriage return isn't added via Spark, so I've changed the behavior here too. The previous roll-up is read and written over here, so there shouldn't be any conflicts.